### PR TITLE
Fix Connectors imports and bugs

### PR DESCRIPTION
**Pull request by Jucchan**
1. Moved connectors to examples/common/python
2. Fixed import statements in connectors
3. Fixed bugs in direct model connectors
4. Improved legibility of ethereum connector test output

### CODE REVIEW

File: examples/common/python/connectors/tcf_connector.toml [hunk 1]
```diff
@@ -28,17 +28,17 @@ type = "Ethereum"
...
-direct_registry_contract_address = "0xB033aFeb6205Bcc322aa93a713db65511a447DF3"
+direct_registry_contract_address = "0x9Be28B132aeE1b2c5A1C50529a636cEd807842cd"
 #Deployed contract address of worker registry contract address.
-worker_registry_contract_address = "0xf652A128606Af90C5B80961E8e29aeE8f1299038"
+worker_registry_contract_address = "0xC5cC1624d450AAcb37B2bc859f7d8632133dB041"
 
 #Ethereum account details
-eth_account = "0xDcB780ea30C1041abfcCCE4233c275B9F313eAf4"
+eth_account = "0xb03874AE05dCe37B162dd234d794d0e22FCdEec5"
 #Ethereum account private key should be set in environment variable WALLET_PRIVATE_KEY
```
Comment (Ram-srini) [inline examples/common/python/connectors/tcf_connector.toml]: Since you changed the ethereum account id, did you set the WALLET_PRIVATE_KEY in Dockerfile?

Comment (Ram-srini) [reply examples/common/python/connectors/tcf_connector.toml]: I could see WALLET_PRIVATE_KEY being set in other PR#23

File: examples/common/python/connectors/direct/unit_tests/test_worker_registry_jrpc_impl.py [hunk 2]
```diff
@@ -20,7 +20,7 @@
 import secrets
 import json
 
-from tcf_connector.worker_registry_jrpc_impl import WorkerRegistryJRPCImpl
+from connectors.direct.worker_registry_jrpc_impl import WorkerRegistryJRPCImpl
 from utility.tcf_types import WorkerType, WorkerStatus
```
Comment (Ram-srini) [inline examples/common/python/connectors/direct/unit_tests/test_worker_registry_jrpc_impl.py]: Update the toml file path. It is pointing to old path tcf_connector..

File: examples/common/python/connectors/direct/unit_tests/test_work_order_jrpc_impl.py [hunk 3]
```diff
@@ -22,7 +22,7 @@
 import time
 import base64
 
-from tcf_connector.work_order_jrpc_impl import WorkOrderJRPCImpl
+from connectors.direct.work_order_jrpc_impl import WorkOrderJRPCImpl
```
Comment (Ram-srini) [inline examples/common/python/connectors/direct/unit_tests/test_work_order_jrpc_impl.py]: Update the tcf_connector.toml file path. It is pointing old path.

File: examples/common/python/connectors/direct/unit_tests/test_work_order_encryption_key_jrpc_impl.py [hunk 4]
```diff
@@ -22,7 +22,7 @@
 import secrets
 
 
-from tcf_connector.work_order_encryption_key_jrpc_impl import WorkOrderEncryptionKeyJrpcImpl
+from connectors.direct.work_order_encryption_key_jrpc_impl import WorkOrderEncryptionKeyJrpcImpl
```
Comment (Ram-srini) [inline examples/common/python/connectors/direct/unit_tests/test_work_order_encryption_key_jrpc_impl.py]: Update the tcf_connector.toml file path. It is pointing old path.

**Comment by manojgop** [comment]
Can you squash the commit into one commit (using git rebase -i) and resubmit the PR

File: examples/common/python/connectors/direct/unit_tests/test_worker_registry_jrpc_impl.py (hunk 2 above)
Comment (Jucchan) [reply examples/common/python/connectors/direct/unit_tests/test_worker_registry_jrpc_impl.py]: Done.

File: examples/common/python/connectors/direct/unit_tests/test_work_order_jrpc_impl.py (hunk 3 above)
Comment (Jucchan) [reply examples/common/python/connectors/direct/unit_tests/test_work_order_jrpc_impl.py]: Done.

File: examples/common/python/connectors/direct/unit_tests/test_work_order_encryption_key_jrpc_impl.py (hunk 4 above)
Comment (Jucchan) [reply examples/common/python/connectors/direct/unit_tests/test_work_order_encryption_key_jrpc_impl.py]: Done.

**Comment by Jucchan** [comment]
I have squashed commits. As a side-note: I was able to fix bugs and update test_worker_registry_jrpc_impl.py, but test_work_order_encryption_key_jrpc_impl.py and test_work_order_jrpc_impl.py are significantly outdated and would need to be fixed on a separate occasion.

**Pull request merged**